boards/arm/nucleo_g431kb/doc/index.rst

```diff
@@ -0,0 +1,221 @@
+  - 2x watchdog timers (independent, window)
+  - 2x 16-bit basic timers
+  - SysTick timer
+  - 1x low-power timer
+
+- Up to 86 fast I/Os, most 5 V-tolerant
+- Memories
+
+  - Up to 128 KB single bank Flash, proprietary code readout protection
+  - Up to 22 KB of SRAM including 16 KB with hardware parity check
+
+- Rich analog peripherals (independent supply)
+
+  - 2x 12-bit ADC 5 MSPS, up to 16-bit with hardware oversampling, 200
+    |micro| A/MSPS
+  - 4x 12-bit DAC, low-power sample and hold
+  - 3x operational amplifiers with built-in PGA
+  - 4x ultra-fast rail-to-rail analog comparators
+
+- 16x communication interfaces
+
+  - 1 x FDCAN controller supporting flexible data rate
+  - 3x I2C FM+(1 Mbit/s), SMBus/PMBus
+  - 4x USARTs (ISO 7816, LIN, IrDA, modem)
+  - 1x LPUART
+  - 3x SPIs (2x with multiplexed half duplex I2S interface)
+  - 1x SAI (serial audio interface)
+  - USB 2.0 full-speed interface with LPM and BCD support
+  - IRTIM (Infrared interface)
+  - USB Type-C™ /USB power delivery controller (UCPD)
+
+- 12-channel DMA controller
+- True random number generator (RNG)
+- CRC calculation unit, 96-bit unique ID
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell*
+
+
+More information about STM32G431KB can be found here:
+
+- `STM32G431KB on www.st.com`_
+- `STM32G4 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_g431kb board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| DAC       | on-chip    | dac                                 |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | rtc                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
```

Comment:
Hi @mariucker ,

There is a new table model. I would recommend you to update to that one. See [stm32l1_disco](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/stm32l1_disco/doc/index.rst) as example.

Reply:
There are > 200 boards using this kind of table. Doesn't feel fair to request a change on this.